pilot/pkg/bootstrap/server.go

```diff
@@ -213,7 +222,7 @@ func NewServer(args PilotArgs) (*Server, error) {
 	if err := s.initMonitor(&args); err != nil {
 		return nil, err
 	}
-	if err := s.initMultiClusterController(&args); err != nil {
```

Comment:
why change this name? It is not only SecretController

Reply:
It is only a secret controller.   So the name change is more accurate.  The new library is essentially a pure secret controller.  It just watches secrets for adds or removes of remote clusters.  It it is up to the add and delete functions in this file what to do when a remote cluster is added or deleted.